README.md

```diff
@@ -1,32 +1,91 @@
-WP-CLI
-======
+# WP-CLI
```

Comment:
Let's keep this as:

```
WP-CLI
======
```

It will be easier to parse this out to turn it into the index.